db/seeds.rb

```diff
@@ -204,6 +204,7 @@
+  {echo_id: "C1000000320-LPDAAC_ECS", has_browseable_granules: false, has_granules: true, browseable_granule: nil, granule: nil, searchable_attributes: {"attributes"=>[{"name"=>"CorrectionAchieved", "data_type"=>"STRING", "description"=>"The identifier to inform the user of the level of correction achieved for the product. (The choices are: Terrain+Precision, Terrain+Systematic, Systematic, Precision)"}]}, orbit: {}},
```

Comment:
This is not the full list of attributes, and it'll be overwritten by the full list of attributes in a background job (except that there's a bug that I just discovered and am now fixing that has been preventing that background job from working, push coming soon).